libkbfs/md_journal.go

```diff
@@ -780,6 +928,21 @@ func (j *mdJournal) put(
 		}
 	}()
 
+	if extra == nil {
```

Comment:
It shouldn't. In that case, both the wkbID and rkbID should be `nil`, and `getExtraMetadata` will then return `nil` without an error.